index 906c8f48039..79f96a3682f 100644
--- a/vendor/sylius/sylius/src/Sylius/Component/Core/Model/Taxon.php
+++ b/vendor/sylius/sylius/src/Sylius/Component/Core/Model/Taxon.php
@@ -15,11 +15,12 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Comparable;
 use Sylius\Component\Resource\Model\TimestampableTrait;
 use Sylius\Component\Taxonomy\Model\Taxon as BaseTaxon;
 use Sylius\Component\Taxonomy\Model\TaxonTranslation;
 
-class Taxon extends BaseTaxon implements TaxonInterface
+class Taxon extends BaseTaxon implements TaxonInterface, Comparable
 {
     use TimestampableTrait;
 
@@ -101,4 +102,12 @@ public static function getTranslationClass(): string
     {
         return TaxonTranslation::class;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function compareTo($other): int
+    {
+        return $this->code === $other->getCode() ? 0 : 1;
+    }
 }
